app/services/connector.js

```diff
@@ -76,6 +76,31 @@ ConnectorService.prototype.getConfig = function getConfig(done) {
   });
 };
 
+ConnectorService.prototype.setCredentials = function setCredentials(user, credentials, done) {
```

Comment:
In order to avoid code repetition and removing compatibility, use the operation `setCloudConfig ` to configure the cloud settings and create a similar to the fog. Notice we're adding the settings to the 'KNOT_CLOUD' only.

Reply:
Done!